packages/cozy-pouch-link/src/PouchManager.spec.js

```diff
@@ -334,4 +471,51 @@ describe('PouchManager', () => {
       }
     })
   })
+
+  describe('warmupQueries', () => {
+    let manager
+    const executeMock = jest.fn()
+    beforeEach(() => {
+      let newManagerOptions = {
+        ...managerOptions,
+        executeQuery: executeMock,
+        doctypesReplicationOptions: {
+          'io.cozy.todos': {
+            strategy: 'fromRemote',
+            warmupQueries: [query1(), query2()]
+          }
+        }
+      }
+      manager = new PouchManager(['io.cozy.todos'], newManagerOptions)
+    })
+
+    it('should executes warmeupQueries on the first replicationLoop only', async () => {
```

Comment:
Can you add test for the clearing of the this.warmupQueries entry if there is an (a)sync error in client.executeQuery ?